GameData/StationPartsExpansionRedux/Patches/SSPXR-KeepFit.cfg

```diff
@@ -0,0 +1,39 @@
+// by default, utility spaces are cramped
+@PART[sspx-airlock-25-1,sspx-utility-125-1,sspx-airlock-125-1]:AFTER[KeepFit]
```

Comment:
Can you add a `NEEDS` here so it's not always added?

Reply:
AFTER implies NEEDS.  (Same for BEFORE.)

Reply:
Figured that out after I posted, hence the merge